__tests__/articles.test.js

```diff
@@ -99,3 +99,57 @@ describe("/api/articles", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    describe("Status 200", () => {
+      it("should respond with an array of comments for the given article_id", () => {
+        return request(app)
+          .get("/api/articles/1/comments")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.comments).toBeInstanceOf(Array);
```

Comment:
This is only checking the first comment. Have a go at looping through the comments here to check that every comment has the property you expect